netci.groovy

```diff
@@ -205,14 +205,14 @@ branchList.each { branchName ->
             if (osGroupMap[os] == 'Windows_NT') {
                 newJob.with {
                     steps {
-                        batchFile("build.cmd /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroupMap[os]} /p:WithCategories=OuterLoop /p:ServiceUri=%WcfServiceUri%")
+                        batchFile("build.cmd /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroupMap[os]} /p:WithCategories=OuterLoop /p:ServiceUri=%WcfServiceUri% /p:BuildType:CI")
```

Comment:
Maybe it's the same but just to confirm... should it be `p:BuildType:CI` or `p:BuildType=CI`? The latter format is used for all other parameters.